### ISSUE TITLE
Unable to run stateful functions

### ISSUE THREAD

**Issue by Lanayx** [kafkaesque-io/pulsar-helm-chart#16]
Is it possible to run stateful functions with your chart? Looks like it is not possible when using `functionsWorkerEnabled: yes` so i tried enabling standalone functions worker and standalone state, but state installation fails on [this line](https://github.com/kafkaesque-io/pulsar-helm-chart/blob/master/helm-chart-sources/pulsar/templates/state-statefulset.yaml#L101), then if I specify standard pulsar image, the statefulset fails with error `org.apache.pulsar.PulsarStandaloneStarter - Unknown option: --stream-containers`

**Comment by cdbartholomew** [comment]
@Lanayx standalone statestorage only works with a custom image we built. It adds some additional options to the Pulsar standalone functionality to enable a properly functioning statestorage. You are welcome to try it:
```
  stateStorage:
    repository: kafkaesqueio/pulsar-all
    pullPolicy: IfNotPresent
    tag: 2.4.2-k2.1
```

We did this because statestorage is new and we wanted to keep the functionality separated from the core bookkeeper nodes.

That being said, you should be able to run with statestorage enabled on the bookies using the Helm chart with just a couple of modifications.

Change the state URL in the to point to bookeeper in `function-configmap.yaml`, like this:

```
stateStorageServiceUrl: "bk://{{ template "pulsar.fullname" . }}-{{ .Values.bookkeeper.component }}:4181"
```


And then in your `values.yaml` file enable state storage on bookkeeper by adding the following setting to the `configData`:

```
    PULSAR_PREFIX_extraServerComponents: org.apache.bookkeeper.stream.server.StreamStorageLifecycleComponent
```

We would welcome a small pull request that properly sets the function worker configmap when statestorage is enabled but standalone is disabled along with some comments in the values.yaml on how to set things properly for this case.

**Comment by Lanayx** [comment]
I've tried the second option and while function worker pod seems to be working fine, I'm unable to run any functions commands, I get this error.
```
Function worker service is not done initializing. Please try again in a little while.

Reason: HTTP 503 Service Unavailable
```
I'm running commands from bastion pod which has _webServiceUrl_ configured to point to proxy pod

**Comment by cdbartholomew** [comment]
Forwarding to the function worker in the proxy is broken in 2.5.0. We have published a 2.5.0 version with a fix: 
```
proxy:
    repository: kafkaesqueio/pulsar-all
    pullPolicy: IfNotPresent
    tag: 2.5.0-kesque-1.1
```

Try changing your _webServiceUrl_ to point to the function worker port 6750.

**Comment by Lanayx** [comment]
Adjusting images solves the issue
